compiler/apps/playground/__tests__/e2e/page.spec.ts

```diff
@@ -55,7 +55,7 @@ const TestComponent2 = ({ x }) => {
 };`,
   },
   {
-    name: 'function-scope-beats-module-scope',
+    name: 'todo-function-scope-does-not-beat-module-scope',
```

Comment:
See attached snap fixture -- playground and the babel plugin diverged before this PR